test_tf.py

```diff
@@ -0,0 +1,153 @@
+import os
+import subprocess
+import unittest
+from main import get_policy_check_type, get_required_inputs, get_optional_inputs, get_sub_command
+from main import get_treat_findings_as_non_blocking_flag, build_command, execute_command, set_output
+
+CLI_POLICY_VALIDATOR = "tf-policy-validator"
+
+POLICY_CHECK_TYPE = "INPUT_POLICY-CHECK-TYPE"
+
+NOT_VALIDATE_POLICY = "NOT_VALIDATE_POLICY"
+VALIDATE_POLICY = "VALIDATE_POLICY"
```

Comment:
should we import variables from [main.py](https://github.com/hlijuen/terraform-aws-iam-policy-validator/blob/8212de2eef5935ac8cde9f4e9fa6578fb7beeb81/main.py#L7-L9) instead?

Reply:
will do.